.github/fabricbot.json

```diff
@@ -1,3600 +0,0 @@
-            ],
-            "timezoneOffset": -8
-          },
-          {
-            "weekDay": 4,
-            "hours": [
-              1,
-              4,
-              7,
-              10,
-              13,
-              16,
-              19,
-              22
-            ],
-            "timezoneOffset": -8
-          },
-          {
-            "weekDay": 5,
-            "hours": [
-              1,
-              4,
-              7,
-              10,
-              13,
-              16,
-              19,
-              22
-            ],
-            "timezoneOffset": -8
-          },
-          {
-            "weekDay": 6,
-            "hours": [
-              1,
-              4,
-              7,
-              10,
-              13,
-              16,
-              19,
-              22
-            ],
-            "timezoneOffset": -8
-          }
-        ],
-        "searchTerms": [
-          {
-            "name": "isPr",
-            "parameters": {}
-          },
-          {
-            "name": "isOpen",
-            "parameters": {}
-          },
-          {
-            "name": "hasLabel",
-            "parameters": {
-              "label": "needs author feedback :memo:"
-            }
-          },
-          {
-            "name": "noActivitySince",
-            "parameters": {
-              "days": 15
-            }
-          },
-          {
-            "name": "noLabel",
-            "parameters": {
-              "label": "no-recent-activity :chart_with_downwards_trend:"
-            }
-          }
-        ],
-        "actions": [
-          {
-            "name": "addLabel",
-            "parameters": {
-              "label": "no-recent-activity :chart_with_downwards_trend:"
-            }
-          },
-          {
-            "name": "addReply",
-            "parameters": {
-              "comment": "This pull request has been automatically marked as stale because it has been marked as requiring author feedback but has not had any activity for **15 days**. It will be closed if no further activity occurs **within 30 days of this comment**."
-            }
-          }
-        ]
-      },
-      "id": "m0PTbzkc0_"
-    },
-    {
-      "taskType": "trigger",
-      "capabilityId": "AutoMerge",
-      "subCapability": "AutoMerge",
-      "version": "1.0",
-      "config": {
-        "taskName": "Automatically merge pull requests",
-        "label": "auto merge :zap:",
-        "silentMode": false,
-        "minMinutesOpen": "480",
-        "mergeType": "merge",
-        "allowAutoMergeInstructionsWithoutLabel": false,
-        "deleteBranches": true,
-        "enforceDMPAsStatus": true,
-        "requireAllStatuses": false,
-        "requireAllStatuses_exemptList": [],
-        "removeLabelOnPush": true,
-        "conditionalMergeTypes": [],
-        "requireSpecificCheckRuns": false,
-        "usePrDescriptionAsCommitMessage": true,
-        "requireSpecificCheckRunsList": [
```

Comment:
@JohannesLampel The previous configuration had a specific check runs list set up for auto-merge. I don't see that replicated in the new configuration.